pkg/logs/input/kubernetes/file_system.go

```diff
@@ -0,0 +1,129 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build kubelet
+
+package kubernetes
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/fsnotify/fsnotify"
+
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/kubelet"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// FileSystem looks for new and deleted pods listening to file system events.
```

Comment:
Don't we want to rely on autodiscovery for this? I thought it was the point of AD